integration-cli/docker_cli_logs_test.go

```diff
@@ -178,6 +178,41 @@ func (s *DockerSuite) TestLogsSince(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestLogsUntil(c *check.C) {
```

Comment:
@jamiehannaford as the `cli` has moved away, we should write `api` integration tests instead of `cli`.  You can take a look at `docker_api_logs_test.go` for inspiration